templates/boot/generic-barebox-tftp-ramdisk-boot-template.jinja2

```diff
@@ -0,0 +1,20 @@
+{% extends 'base/kernel-ci-base-tftp-deploy.jinja2' %}
+{% block metadata %}
```

Comment:
I'm not sure if this is actually needed, if extend a template but don't change the contents of a block it should leave it as in the parent template.